apps/address-service/domains/common/utils/services/suggest/SuggestionKeystoneApp.js

```diff
@@ -18,37 +18,55 @@ class SuggestionKeystoneApp {
             next()
         }
 
-        app.get('/suggest', setNoCache, async (req, res, next) => {
+        app.all('/suggest', setNoCache, async (req, res, next) => {
+            if (!['GET', 'POST'].includes(req.method)) {
```

Comment:
there is no need to create static array `['GET', 'POST']` on each request, better to move it to a constant

Reply:
Done